pom.xml

```diff
@@ -370,33 +367,17 @@
                         <artifactId>maven-surefire-plugin</artifactId>
                         <version>${surefire.version}</version>
                         <configuration>
-                            <!--  If enabled, will run the GraalVM tracing agent while running tests  -->
-                            <!--              <argLine>-agentlib:native-image-agent=config-output-dir=META-INF/native-image</argLine>-->
```

Comment:
don't we need that anymore?

Reply:
You can run the agent by adding some configuration to the `native-maven-plugin`: https://graalvm.github.io/native-build-tools/latest/maven-plugin.html#agent-support-enabling
However just running the agent creates quite a bit config that's not required at all, so it's not that great. Plus the only config that you need are the JNI calls in `JNI_OnLoad` and some resources. That should be straightforward enough without the agent.